<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Counter</class>
 <widget class="QWidget" name="Counter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counter</string>
  </property>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="restart">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RESTART</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>29</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
